modeldb/modelrun.py

```diff
@@ -134,7 +135,25 @@ def clean_model_dir(model):
 
 
 def compile_mods(model, mods):
-    run_commands(model, ["nrnivmodl"] + mods, work_dir=model.run_info["start_dir"])
+    # Unfortunately nrnivmodl doesn't have an option to steer how much build
```

Comment:
just a note: `nrnivmodl-core` has such option. May be we could add back port it in nrnivmodl.